service-front/module/Application/src/Contracts/OpgApiServiceInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Contracts;
+
+interface OpgApiServiceInterface
```

Comment:
FWIW we don't tend to use this pattern when interfaces only have one concrete. Which isn't to say "don't use it" but more "don't feel obliged to use it".